test/spec/format.spec.js

```diff
@@ -1,62 +1,76 @@
-import { format, time } from '../../src/js/format';
+import { time } from '../../src/js/format';
 
 describe('Format', () => {
     /** @type {import('sinon').SinonSandbox} */
     let sandbox;
 
-    /** @type {string} */
-    let localeOverride;
+    [
```

Comment:
I'm surprised not to see any additions here! Is there a test that would have failed prior to the change?

Reply:
Good call, I meant to actually add the case from the issue! I was focused on getting the setup/te~~e~~ardown switched over and forgot.